plugins/ExtendedTools/objmgr.c

```diff
@@ -877,35 +2397,53 @@ INT_PTR CALLBACK WinObjDlgProc(
                 EtObjectManagerRootDirectoryObject
                 );
 
-            {
```

Comment:
Hi, why this block was removed please?

9ef7295's description doesn't explain it and there seem to be no related comments.

It was [added](https://github.com/winsiderss/systeminformer/commit/b84bdc9a9b4b5761757ca5e46764b1021d67e475#diff-dbc6ac39b248480edbffe8a3f3525e4b0dee98edc4fd9ceeb901bf3726e52ab7R880-R889) in b84bdc9a9b4b5761757ca5e46764b1021d67e475, implementing #2074.

Reply:
I could removed this by mistake since many things were rewritten and added. I will look into and try to revert this and commit to #2284.

Reply:
fixes in 165f3a55372a629b8fcbce8d620f0191c2e54e25